content/en/news/releases/1.6.x/announcing-1.6/change-notes/index.md

```diff
@@ -6,43 +6,40 @@ weight: 10
 
 ## Improvements
-- ***Improved*** support for [Kubernetes Ingress](https://preliminary.istio.io/docs/tasks/traffic-management/ingress/kubernetes-ingress/), adding support for reading certificates from Secrets, `pathType`, and IngressClass.
+- ***Improved*** support for [Kubernetes Ingress](/docs/tasks/traffic-management/ingress/kubernetes-ingress/), adding support for reading certificates from Secrets, `pathType`, and IngressClass.
 - ***Added*** a new `proxy.istio.io/config` annotation to override proxy configuration per pod.
-- ***Added*** support for using appProtocol to select the [protocol for a port](https://preliminary.istio.io/docs/ops/configuration/traffic-management/protocol-selection/) for Kubernetes 1.18+.
-- SDS for mixer? Not sure we need this or its an implementation detail
-- ***Improved*** Prometheus integration experience by adding standard Prometheus scrape annotations to proxies and the control plane workloads. This removes the need for specialized configuration to discover and consume Istio metrics. More details are available in the [operations guide for Prometheus](https://preliminary.istio.io/docs/ops/integrations/prometheus#option-2-metrics-merging).
-- ***Updated*** default Telemetry V2 configuration to avoid using host header to extract destination service name at gateway. This prevents unbound cardinality due to untrusted host header and implies that destination service labels are going to be omitted for request hits Blackhole and Passthrough at gateway.
+- ***Added*** support for using appProtocol to select the [protocol for a port](/docs/ops/configuration/traffic-management/protocol-selection/) for Kubernetes 1.18+.
+- ***Improved*** Prometheus integration experience by adding standard Prometheus scrape annotations to proxies and the control plane workloads. This removes the need for specialized configuration to discover and consume Istio metrics. More details are available in the [operations guide for Prometheus](/docs/ops/integrations/prometheus#option-2-metrics-merging).
+- ***Updated*** default telemetry v2 configuration to avoid using host header to extract destination service name at gateway. This prevents unbound cardinality due to untrusted host header and implies that destination service labels are going to be omitted for request hits blackhole and passthrough at gateway.
 - ***Improved*** support for [customizing v2 metrics generation](https://istio.io/docs/tasks/observability/metrics/customize-metrics/). This allows mesh operators to add and remove labels used in Istio metrics based on expressions over the set of available request and response attributes.
-- ***Improved*** the output of the istioctl install command.
-- ***Added*** a new command, istioctl install. This is an alias for manifest apply.
-
-## Installation Impacts
+- ***Improved*** the output of the istioctl install command (e.g. more information is displayed to the user, use of color)
```

Comment:
gah - this is an awesome feature. Either surprise the operator with it, or communicate the exact change.

Something that is a good surprise:

`**Improved** the display characteristics of the `istioctl` command. Surprise awaits!`

That may be too corny...

Reply:
Let's try your suggestion and see how it plays itself out.